src/set_theory/surreal.lean

```diff
@@ -315,58 +315,47 @@ lift₂ (λ x y _ _, x < y) (λ x₁ y₁ x₂ y₂ _ _ _ _ hx hy, propext (lt_c
-{ le_antisymm := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩ h₁ h₂; exact quot.sound ⟨h₁, h₂⟩,
-  ..surreal.preorder }
-
-noncomputable instance : linear_order surreal :=
-{ le_total := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical; exact
-    or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
-  decidable_le := classical.dec_rel _,
-  ..surreal.partial_order }
-
 /-- Addition on surreals is inherited from pre-game addition:
 the sum of `x = {xL | xR}` and `y = {yL | yR}` is `{xL + y, x + yL | xR + y, x + yR}`. -/
 def add : surreal → surreal → surreal :=
 surreal.lift₂
   (λ (x y : pgame) (ox) (oy), ⟦⟨x + y, numeric_add ox oy⟩⟧)
-  (λ x₁ y₁ x₂ y₂ _ _ _ _ hx hy, quot.sound (pgame.add_congr hx hy))
+  (λ x₁ y₁ x₂ y₂ _ _ _ _ hx hy, quotient.sound (pgame.add_congr hx hy))
 
 instance : has_add surreal := ⟨add⟩
 
-theorem add_assoc : ∀ (x y z : surreal), (x + y) + z = x + (y + z) :=
-begin
-  rintros ⟨x⟩ ⟨y⟩ ⟨z⟩,
-  apply quot.sound,
-  exact add_assoc_equiv,
-end
-
-theorem zero_add : ∀ (x : surreal), 0 + x = x :=
-by { rintro ⟨x, ox⟩, exact quotient.sound (pgame.zero_add_equiv _) }
-
-theorem add_zero : ∀ (x : surreal), x + 0 = x :=
-by { rintro ⟨x, ox⟩, exact quotient.sound (pgame.add_zero_equiv _) }
-
-instance : add_monoid surreal :=
-{ add       := surreal.add,
-  add_assoc := surreal.add_assoc,
-  zero      := 0,
-  zero_add  := surreal.zero_add,
-  add_zero  := surreal.add_zero }
+def neg : surreal → surreal :=
+surreal.lift
+  (λ x ox, ⟦⟨-x, pgame.numeric_neg ox⟩⟧)
+  (λ _ _ _ _ a, quotient.sound (pgame.neg_congr a))
+
+instance : ordered_add_comm_group surreal :=
+{ add               := surreal.add,
+  add_assoc         := by { rintros ⟨_⟩ ⟨_⟩ ⟨_⟩, exact quotient.sound add_assoc_equiv },
+  zero              := 0,
+  zero_add          := by { rintros ⟨_⟩, exact quotient.sound (pgame.zero_add_equiv _) },
+  add_zero          := by { rintros ⟨_⟩, exact quotient.sound (pgame.add_zero_equiv _) }, 
+  neg               := surreal.neg, 
+  sub               := λ x y, x + surreal.neg y,
+  sub_eq_add_neg    := by try_refl_tac,
+  add_left_neg      := by { rintros ⟨_⟩, exact quotient.sound pgame.add_left_neg_equiv }, 
+  add_comm          := by { rintros ⟨_⟩ ⟨_⟩, exact quotient.sound pgame.add_comm_equiv },
+  le                := surreal.le,
+  lt                := surreal.lt,
+  le_refl           := by { rintros ⟨_⟩, refl },
+  le_trans          := by { rintros ⟨_⟩ ⟨_⟩ ⟨_⟩, exact pgame.le_trans },
+  lt_iff_le_not_le  := by { rintros ⟨_, ox⟩ ⟨_, oy⟩, exact pgame.lt_iff_le_not_le ox oy },
+  le_antisymm       := by { rintros ⟨_⟩ ⟨_⟩ h₁ h₂, exact quotient.sound ⟨h₁, h₂⟩ },
+  add_le_add_left   := by { rintros ⟨_⟩ ⟨_⟩ hx ⟨_⟩, exact pgame.add_le_add_left hx } }
+  
+noncomputable instance : linear_order surreal :=
+{ le_total := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical; exact
+    or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
+  decidable_le := classical.dec_rel _,
+  ..(infer_instance : partial_order surreal) }
```

Comment:
There's a stronger isntance you can use here:
```suggestion
noncomputable instance : linear_ordered_add_comm_group surreal :=
{ le_total := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical; exact
    or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
  decidable_le := classical.dec_rel _,
  ..surreal.ordered_add_comm_group }
```
It's worth keeping separate from the previous instance though since this one is non-computable, but that one is computable.